### PR TITLE
DPI scale

### PR DESCRIPTION
**Pull request by keenfoong**
Dpi scale using pyside libraries only.

Plus a windowFlag helper function which prints out the active flags for a window.

### CODE REVIEW

File: zoo/libs/pyqt/utils.py [hunk 1]
```diff
@@ -157,3 +157,96 @@ def updateStyle(widget):
     """
     widget.setStyle(widget.style())
 
+<<<<<<< HEAD
```
Comment (dsparrow27) [inline zoo/libs/pyqt/utils.py]: I guess you forgot to remove this?

File: zoo/libs/pyqt/utils.py [hunk 2]
```diff
@@ -157,3 +157,93 @@ def updateStyle(widget):
...
+    flag_type = (windowFlags & QtCore.Qt.WindowType_Mask)
+
+    if flag_type == QtCore.Qt.Window:
+        text = "QtCore.Qt.Window"
+    elif flag_type == QtCore.Qt.Dialog:
+        text = "QtCore.Qt.Dialog"
+    elif flag_type == QtCore.Qt.Sheet:
+        text = "QtCore.Qt.Sheet"
+    elif flag_type == QtCore.Qt.Drawer:
+        text = "QtCore.Qt.Drawer"
```
Comment (dsparrow27) [inline zoo/libs/pyqt/utils.py]: Can you not do
`str(flagType)`
which would come out as "PySide.QtCore.Qt.WindowType.Window" or PySide.QtCore.Qt.WindowType.Dialog

File: zoo/libs/pyqt/utils.py (hunk 1 above)
Comment (keenfoong) [reply zoo/libs/pyqt/utils.py]: yep oops. This is removed.

File: zoo/libs/pyqt/utils.py (hunk 2 above)
Comment (keenfoong) [reply zoo/libs/pyqt/utils.py]: str(flagType) prints out "`<WindowFlags object at 0x00000167BDB0B9F0>`" on my end. Eg if 
I run 
```python
print((w.windowFlags() & QtCore.Qt.WindowType_Mask))
```

It gives this `<WindowFlags object at 0x00000167BDB0B9F0>`

But I think I can clean up this code a bit, I'll do that and push up the changes.

File: zoo/libs/pyqt/utils.py [hunk 3]
```diff
@@ -157,3 +157,82 @@ def updateStyle(widget):
     """
     widget.setStyle(widget.style())
 
+
+def windowFlagsString(windowFlags):
+    """Returns a nice string that describes whats inside a windowFlags object
+
+    .. code-block:: python
```
Comment (dsparrow27) [inline zoo/libs/pyqt/utils.py]: Must have a space after 
.. code-block:: python

File: zoo/libs/pyqt/utils.py [hunk 4]
```diff
@@ -157,3 +157,82 @@ def updateStyle(widget):
...
+        >>> print(windowFlagsString(self.windowFlags()))
```
Comment (dsparrow27) [inline zoo/libs/pyqt/utils.py]: Remove 
>>>

It's easier it terms of copy pasting from the docs for the user

File: zoo/libs/pyqt/utils.py [hunk 5]
```diff
@@ -157,3 +157,82 @@ def updateStyle(widget):
...
+
+    Prints out:
+    .. code-block:: python
```
Comment (dsparrow27) [inline zoo/libs/pyqt/utils.py]: Space

File: zoo/libs/pyqt/utils.py [hunk 6]
```diff
@@ -157,3 +157,82 @@ def updateStyle(widget):
...
+    return text
+
+
+def dpiScale(value):
+    """Resize by value based on current DPI
+
+    :param value:
+    :return:
+    """
+    DEFAULT_DPI = 96
```
Comment (dsparrow27) [inline zoo/libs/pyqt/utils.py]: Move constant to the pyqt constants module

**Comment by keenfoong** [comment]
Ok cool, I think I'll get on the documenting stuff after I've finished all the little bugs for zootoolsets.

**Pull request merged**